README.md

```diff
@@ -1,2 +1,62 @@
-# tslint-plugin-shopify
-Shopify’s Typescript rules and configs.
+# tslint-config-shopify
+Shopify’s TSlint rules and configs.
+
+## Installation 
+
+Install [TSlint](https://palantir.github.io/tslint/) and `tslint-config-shopify` : 
+
+**With Yarn**
+``` 
+yarn add --dev tslint tslint-config-shopify
+```
+
+**With npm**
+``` 
+npm install tslint tslint-config-shopify --save-dev
+```
+
+
+## Usage
+Shopify’s TSLint rules come bundled in `tslint-config-shopify`. 
+To enable these rules, create a `tslint.json` file at the root level of your project, and extend `tslint-config-shopify`. 
+```
+{
+  "extends": "tslint-config-shopify"
+}
+```
+
+Now you can run TSLint by adding the following linting script to your `package.json`. See [here](https://palantir.github.io/tslint/usage/cli/) for more script configurations.
+```
+{
+  "scripts": {
+    "tslint": "tslint -c tslint.json './{app,server,client,packages,config}/**/*.{ts,tsx}' --exclude './**/node_modules/**/*.{ts,tsx}' --project tsconfig.json --type-check"
```

Comment:
@GoodForOneFare this script is a bit more specific to our team.
However, your right. For sample code, it should be simpler. I'll update it.